Source/Fuse.Controls.Panels/TreeRenderer.iOS.uno

```diff
@@ -86,7 +86,11 @@ namespace Fuse.Controls
 
 		void ITreeRenderer.Placed(Element e)
 		{
-			_elements[e].SetSize(e.ActualSize);
+			var viewHandle = _elements[e];
+			if (viewHandle.NeedsRenderBounds)
+				viewHandle.SetSizeAndVisualBounds(e.ActualSize, e.RenderBoundsWithoutEffects);
```

Comment:
I am using the renderbounds to position a `CALayer` inside a `UIView`, so as long as the Rect has a valid position it should be fine